server.js

```diff
@@ -18,31 +18,30 @@ var secondsLeft = 0;
 var currentQuestion = {question: '', id: ''};
 var maxAnswers = 3;
 
-
 app.get('/', function(req, res){
   res.render('index');
 });
 
 socket.on('connection', function(client) {
-  
+
   client.on('join', function() {
-    console.log('User with id ' + client.id + " connected")
+    console.log('User with id ' + client.id + " connected");
     clients[client.id] = {upvoted: []};
     client.emit('join', {id: client.id, answers: answers, question: currentQuestion});
     console.log(clients);
     socket.emit('clients online', Object.keys(clients).length);
   });
-  
+
   client.on('disconnect', function() {
-    console.log('User ' + client.id + " disconnected")
+    console.log('User ' + client.id + " disconnected");
```

Comment:
Added numerous missing semicolons. JSHint (http://jshint.com) actually lets you know about minor javascript warnings/errors like missing semicolons. My text editor,  Sublime Text, even has a JSHint plugin. If your text editor supports plugins, I'd consider looking into it. Easy way to improve your code quality just a tad. We all miss semicolons from time to time.